### DISCUSSION TITLE
How to downgrade to pytorch geometric 1.6.0?

### DISCUSSION THREAD

**Discussion by rutopio** [pyg-team/pytorch_geometric#3281] [Q&A]
Due to some deprecated code that I used, I want to downgrade `torch-geometric` to version `1.6.0`. 

However, after trying

```$ pip uninstall torch-geometric```
```$ pip install torch-geometric==1.6.0```

I got some problem with the `torch-scatter`, `torch-sparse`, `torch-cluster`, `torch-spline-conv`. 

They are the latest version (match for `torch-geometric==2.0.1`) but not suitable with `torch-geometric` in `1.6.0`, making the packages get error.

Does someone know the match version of each one, for instance, `torch_cluster-1.5.9` (I'm not sure)? 

Thanks for any help.

**Comment by rusty1s** [comment]
What are the errors you run into? I think that all latest versions of `torch-*` should be compatible with PyG 1.6.0. For PyG 1.6.0, these are the minimum version requirements.:
```
torch-scatter>=2.0.5
torch-sparse>=0.6.4
torch-cluster>=1.5.6
torch-spline-conv>=1.2.0
```
